packages/syft/src/syft/core/node/common/client_manager/user_api.py

```diff
@@ -49,6 +49,11 @@ def __delitem__(self, key: int) -> None:
         self.delete(user_id=key)
 
     def create(self, **kwargs: Any) -> None:
+        budget = kwargs.get("budget")
```

Comment:
Q: @rasswanth-s  probably this should be deleted? (since this is still on the client side and they were already verified in `domain_client`?)

Reply:
I think we could have it here also george, as we expose the client side API , of domain_client.users.create in addition to domin_client.create_user.

Reply:
Done! I made the changes locally, but I forgot to add the `test` directory to the commit :facepalm: